tools/clang/unittests/HLSLExec/ExecutionTest.cpp

```diff
@@ -417,9 +417,8 @@ class ExecutionTest {
 
   dxc::DxCompilerDllLoader m_support;
 
+  std::optional<D3D12SDKSelector> D3D12SDK;
```

Comment:
nit: If we're going to avoid hugarian notation on this new member I think it would also make sense to update m_support and m_D3DInitCompleted to not use it anymore either.